packages/compass-components/src/components/file-input.tsx

```diff
@@ -194,7 +195,7 @@ function FileInput({
     );
   };
 
-  const applyTheme = global?.process?.env?.COMPASS_LG_DARKMODE === 'true';
+  const applyTheme = preferences.getPreferences().lgDarkmode;
```

Comment:
Maurizio and I had a conversation recently about the idea of moving feature flags to also be communicated as preferences recently, lmk if y'all have any thoughts on it.

Reply:
I think it would be worth it if we ever consider exposing these to the user (like github feature preview for example), otherwise I would probably prefer them to be just env vars to distinct them from the actual options. But also don't really have a very strong opinion here, wouldn't mind if you feel like it's a good idea to move them to prefs

Reply:
I don't have super strong opinions either, but I do like that it makes it potentially a bit easier to ask users to try something out and that it provides us with a single mechanism for reading configurable properties of the application

Reply:
Ahh, okay, so those will not be exposed in the UI, but you'd still be able to set them with command args, yeah, this makes sense